src/index.html

```diff
@@ -9,6 +9,24 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header_logo">
+        <a href="index.html">
+          <img src="/src/images/logo.png" alt="MOYO">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item"><a class="nav_link is-active" href="#">Apple</a></li>
```

Comment:
there shouldn't be more than one tag in one line, fix it everywhere
```suggestion
          <li class="nav_item">
            <a class="nav_link is-active" href="#">Apple</a>
          </li>
```